.github/workflows/ubuntu.yml

```diff
@@ -0,0 +1,23 @@
+name: Run all tests on CPU (GitHub-hosted)
+
+on: push
+
+jobs:
+  run_tests_torch_and_tf:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        pip install .[sklearn,tf,torch,testing]
+
+    - name: Run all tests on CPU
+      env:
+        OMP_NUM_THREADS: 1
+      run: |
+        python -m pytest -n 8 --dist=loadfile -s -v ./tests/
```

Comment:
FYI with @LysandreJik we suppose TF QA pipeline is failing in some cases because of too many parallel runs leading to OOM on the instance itself.

I've open a branch about this one, launching only 4 parallel works `tf_qa_pipelines_test`.